src/app/digital/internal/sim/Signal.ts

```diff
@@ -0,0 +1,43 @@
+
+
+export enum Signal {
+    Off        =  0,
+    On         =  1,
+    Metastable = -1,
```

Comment:
Not sure if I'm misunderstanding the terminology, but wouldn't this actually be the `MetaUnstable` case?

Reply:
It's just called Metastability, that's just the term I believe